rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2205,4 +2244,11 @@ private List<ECKey> getMstPublicKeysFromFederationMembers(List<FederationMember>
             .map(FederationMember::getMstPublicKey)
             .collect(Collectors.toList());
     }
+
+    private Transaction getTransactionFromCaller(RskAddress authorizer) {
```

Comment:
We have this in `TransactionUtils` class

Reply:
Updated.